src/js/player.js

```diff
@@ -2315,17 +2312,22 @@ class Player extends Component {
           this.error({ code: 4, message: this.localize(this.options_.notSupportedMessage) });
         }, 0);
 
-        // we could not find an appropriate tech, but let's still notify the delegate that this is it
-        // this needs a better comment about why this is needed
+        // we could not find an appropriate tech, but let's still notify the
+        // delegate that this is it this needs a better comment about why this
```

Comment:
can you revert this change or put a new line in? Otherwise the two statements run in together.

Reply:
Ah, yes, it is hard to distinguish sentences without capitals and punctuation. ;)